src/tools/rectify-path

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+#  rectify-path
+#  Copyright 2017 Edward Wang <edward.c.wang@compdigitec.com>
+#
+#  Rectify recursively all the given keys which end in a given suffix (default
+#  "_path").
+
+# pylint: disable=invalid-name
+
+from __future__ import print_function
+
+import argparse
+import json
+import sys
+
+# Python 2 vs 3, mumbo jumbo
```

Comment:
It's written in Python 2 but made executable in Python 3 for future proofing, when we do make Python 3 mandatory. Mainly just some forward thinking to save me a little time later.